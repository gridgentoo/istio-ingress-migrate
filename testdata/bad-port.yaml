apiVersion: v1
items:
- apiVersion: extensions/v1beta1
  kind: Ingress
  metadata:
    annotations:
      kubernetes.io/ingress.class: istio
    name: http-only
    namespace: default
  spec:
    rules:
    - host: foo.bar.not-matched.com
      http:
        paths:
        - backend:
            serviceName: echo
            servicePort: 80
          path: /*
          pathType: ImplementationSpecific
- apiVersion: networking.istio.io/v1alpha3
  kind: Gateway
  metadata:
    name: istio-autogenerated-k8s-ingress
    namespace: gke-system
  spec:
    selector:
      istio: ingress-gke-system
    servers:
    - hosts:
      - '*'
      port:
        name: http
        number: 1234
        protocol: HTTP2
kind: List
metadata:
  resourceVersion: ""
  selfLink: ""

