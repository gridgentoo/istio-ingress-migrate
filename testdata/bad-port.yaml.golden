ERR01: Gateway istio-autogenerated-k8s-ingress server for hosts [*] has unsupported port: 1234
ERR06: failed to find a matching HTTPS credential for http-only/foo.bar.not-matched.com; will be HTTP only
---
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  annotations:
    kubernetes.io/ingress.class: istio
  creationTimestamp: null
  name: http-only
  namespace: default
spec:
  rules:
  - host: foo.bar.not-matched.com
    http:
      paths:
      - backend:
          serviceName: echo
          servicePort: 80
        path: /*
        pathType: ImplementationSpecific
status:
  loadBalancer: {}

failures detected during execution
